Bottom: 91e0748f6a40f1f295a0e74ad65dfcdc8448126b
Top:    2039ae95c74a55fb22d95656289bb536ed229a2e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 02:35:25 +0000

Refresh of tag-circular-contours-so-they-

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index d9ded42..1c7ead9 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -183,7 +183,6 @@ void common_fill_pcb_polygon (hidGC gc, PolygonType *poly,
 static int thindraw_hole_cb (PLINE *pl, void *user_data)
 {
   hidGC gc = user_data;
-
   thindraw_contour (gc, pl);
   return 0;
 }
